static/app/views/issueList/header.tsx

```diff
@@ -130,7 +130,7 @@ function IssueListHeader({
         </ButtonBar>
       </Layout.HeaderActions>
       <StyledGlobalEventProcessingAlert projects={selectedProjects} />
-      <StyledTabs selectedKey={query} onSelectionChange={() => {}}>
+      <StyledTabs value={query}>
```

Comment:
This is why we should remove `selectedKey`, `defaultSelectedKey`, etc. — to avoid further confusion.